mne_connectivity/spectral/epochs.py

```diff
@@ -719,6 +729,13 @@ def spectral_connectivity_epochs(
         respectively, using singular value decomposition. If None, the rank of
         the data is computed and projected to. Only used if ``method`` contains
         any of ``['cacoh', 'mic', 'mim', 'gc', 'gc_tr']``.
+    n_components : int
+        Number of connectivity components to extract from the data. If an `int`, the
+        number of components must be <= the minimum rank of the seeds and targets. E.g.
+        if the seed channels had a rank of 5 and the target channels had a rank of 3,
+        ``n_components`` must be <= 3. If `None`, the number of components equal to the
+        minimum rank of the seeds and targets is extracted (see the ``rank`` parameter).
+        Only used if ``method`` contains any of ``['cacoh', 'mic']``.
```

Comment:
Useful to start giving people hints on what version they need to have access to some of this stuff
```suggestion
        Only used if ``method`` contains any of ``['cacoh', 'mic']``.

        .. versionadded:: 0.8.0
```